frontend-react/src/components/Table/ColumnData.tsx

```diff
@@ -101,15 +102,29 @@ export const ColumnData = ({
             if (param) return action(rowData[param]);
             return action();
         };
+        const showActionButton = () => {
+            if (actionButtonHandler && actionButtonParam) {
+                return actionButtonHandler(rowData[actionButtonParam]);
+            } else {
+                return true;
+            }
+        };
+
         return tableData(
-            <Button
-                className="font-mono-2xs line-height-alt-4"
-                type="button"
-                unstyled
-                onClick={doAction}
-            >
-                {displayValue}
-            </Button>
+            <>
+                {showActionButton() ? (
+                    <Button
+                        className="font-mono-2xs line-height-alt-4"
+                        type="button"
+                        unstyled
+                        onClick={doAction}
```

Comment:
can we just have this be `() => action(rowData[param])`?

Reply:
I believe these lines should cover that case: 

https://github.com/CDCgov/prime-reportstream/blob/6221b00d63a07953d504388373255ce90248e23b/frontend-react/src/components/Table/ColumnData.tsx#L97-L99